app/frontend/css/application.scss

```diff
@@ -89,157 +89,175 @@ $govuk-global-styles: true; // https://frontend.design-system.service.gov.uk/sas
   }
 
 //defra header
-.defra-internal-header {
+.defra {
```

Comment:
The only downside to amending the SCSS to be more brief is that it was copied directly from the prototype styling. So if the prototype styling changed, it would be more difficult to transpose any changes into this version of styling - just a thought